tests/testcases/computed-property/expected.d.ts

```diff
@@ -10,4 +10,4 @@ type Klass = {
   [0]: C;
   [Dprop]: D;
 };
-export { Klass };
+export type { Klass };
```

Comment:
I’m a bit surprised why `A, B, C, D` are not exported in this case 🤔 
But this existed before, so not worth taking a look in this PR

Reply:
Should they be? They are not exported in the index.d.ts.